visits-service-new/src/main/java/com/petclinic/visits/visitsservicenew/PresentationLayer/VisitController.java

```diff
@@ -25,6 +33,10 @@ public Mono<ResponseEntity<VisitResponseDTO>> getVisitByVisitId(@PathVariable St
     public Flux<VisitResponseDTO> getVisitByPractitionerId(@PathVariable int practitionerId){
         return visitService.getVisitsForPractitioner(practitionerId);
     }
+    @GetMapping(value="practitioner/{practitionerId}/{month}", produces = MediaType.TEXT_EVENT_STREAM_VALUE)
+    public Flux<VisitResponseDTO> getVisitsByPractitionerIdAndMonth(@PathVariable int practitionerId, @PathVariable int month){
+        return visitService.getVisitsByPractitionerIdAndMonth(practitionerId, month);
+    }
```

Comment:
Method is reactive, was just moved around, could return Flux<ResponseEntity<VisitResponseDTO>> to handle status codes and exceptions, int month could be a query parameter and combined with getVisitByPractitionerId, method seems to work.